spec.html

```diff
@@ -11060,16 +11060,16 @@ <h1>Examples of Automatic Semicolon Insertion</h1>
       <p>In the circumstance that an assignment statement must begin with a left parenthesis, it is a good idea for the programmer to provide an explicit semicolon at the end of the preceding statement rather than to rely on automatic semicolon insertion.</p>
     </emu-clause>
   </emu-clause>
-  <emu-clause id="sec-hazards-of-automatic-semicolon-insertion">
-    <h1>Hazards of Automatic Semicolon Insertion</h1>
+  <emu-clause id="sec-interesting-cases-of-automatic-semicolon-insertion">
+    <h1>Interesting Cases of Automatic Semicolon Insertion</h1>
```

Comment:
One way to talk about it would be "Cases of where semicolons are not inserted"

Reply:
That means something different.  Like, semicolons would not be inserted.
Hazards mean that semicolons could be inserted, but could lead to unintended behavior.

To keep the meaning the same:

Maybe: "Situations of Automatic Semicolon Insertion that could result in unexpected behavior"

Reply:
This section is actually about places where semicolons are not inserted. There is another section about places where semicolons are inserted (which the original patch was not terming a "hazard").